src/plugins/telemetry/server/routes/telemetry_usage_stats.ts

```diff
@@ -24,16 +24,18 @@ export function registerTelemetryUsageStatsRoutes(
       validate: {
         body: schema.object({
           unencrypted: schema.boolean({ defaultValue: false }),
+          refreshCache: schema.boolean({ defaultValue: false }),
```

Comment:
should we update the get example UI (in the Advanced Settings) to always refresh the cache?

Reply:
Great suggestion :+1: i'll update that